src/onnx/CMakeLists.txt

```diff
@@ -37,7 +41,11 @@ set_target_properties(migraphx_onnx PROPERTIES EXPORT_NAME onnx)
 migraphx_generate_export_header(migraphx_onnx)
 rocm_set_soversion(migraphx_onnx ${MIGRAPHX_SO_VERSION})
 rocm_clang_tidy_check(migraphx_onnx)
-target_link_libraries(migraphx_onnx PRIVATE onnx-proto "-Wl,--exclude-libs,ALL")
+if(WIN32)
+    target_link_libraries(migraphx_onnx PRIVATE onnx-proto)
+else()
+    target_link_libraries(migraphx_onnx PRIVATE onnx-proto "-Wl,--exclude-libs,ALL")
+endif()
```

Comment:
You can write it like this since `onnx-proto` is needed for both windows and linux:

```cmake
target_link_libraries(migraphx_onnx PRIVATE onnx-proto)
if(NOT WIN32)
    target_link_libraries(migraphx_onnx PRIVATE "-Wl,--exclude-libs,ALL")
endif()
```